### PR TITLE
Ue4 bp update

### PR DESCRIPTION
**Pull request by cyberblaststudios**
Added blueprint support for JoinRequest() and Respond, added latent
blueprint node for getting Discord profile images

### CODE REVIEW

File: examples/unrealstatus/Plugins/discordrpc/Source/discordrpc/Public/DiscordRpcBlueprint.h [hunk 1]
```diff
@@ -3,19 +3,57 @@
...
+			Username = FString(ANSI_TO_TCHAR(request->username));
+			Avatar = FString(ANSI_TO_TCHAR(request->avatar));
+		}	
+	}
+};
+
+UENUM(BlueprintType)
+enum class EDiscordJoinRequestResponse : uint8
+{
+	DJR_NO = DISCORD_REPLY_NO UMETA(DisplayName = "Yes"),
```
Comment (crmarsh) [inline examples/unrealstatus/Plugins/discordrpc/Source/discordrpc/Public/DiscordRpcBlueprint.h]: copy-n-paste error?

**Comment by crmarsh** [comment]
Neat, thanks for working on this, I'll take a closer look.

File: examples/unrealstatus/Plugins/discordrpc/Source/discordrpc/Public/DiscordRpcBlueprint.h (hunk 1 above)
Comment (cyberblaststudios) [reply examples/unrealstatus/Plugins/discordrpc/Source/discordrpc/Public/DiscordRpcBlueprint.h]: Yeah likely, there was something up with the line endings, I can take a look at it

**Comment by msciotti** [comment]
I'm gonna close this because these methods were added in #125. Thank you for getting the ball rolling for me!

**Pull request closed without merging**